source/_components/homekit.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: "HomeKit"
+description: "Instructions how to setup the HomeKit component in Home Assistant."
+date: 2018-02-20 17:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: HomeKit
```

Comment:
category should be `voice` cause of siri.maybe also `other`

Reply:
Voice would work.